guides/v2.1/cloud/project/project-patch.md

```diff
@@ -1,263 +1,49 @@
 ---
 group: cloud
-title: Patch Magento Commerce (Cloud)
+title: Apply custom patches
 version: 2.1
 github_link: cloud/project/project-patch.md
 functional_areas:
   - Cloud
   - Upgrade
 ---
+Sometimes we provide a custom patch to address a specific issue. Third-party extension developers can also provide a custom patch. Copy the custom patches to the `/m2-hotfixes` directory and test them on your local workstation.
 
-You can apply patches to update {{site.data.var.ece}}. We recommend using a new active branch and Integration environment for applying and testing the patch prior to fully deploying across all environments. We strongly recommend you test patches locally so you can identify and resolve any issues.
+{% include cloud/note-upgrade.md %}
 
-The `composer update` command performs a {{site.data.var.ece}} upgrade with patches and hotfixes. Check the `magento/magento-cloud-configuration/patches` folder for available patches.
+#### To apply and test a custom patch:
+You can only apply patches during the build phase of redeployment.
 
-To upgrade and test {{site.data.var.ece}} (including patches and hotfixes), see [Upgrade Magento Commerce (Cloud)]({{ page.baseurl }}/cloud/project/project-upgrade.html). Starting with 2.2.0, we use `magento/ece-tools` to update {{site.data.var.ece}} with new features and fixes and to update build and deploy processes. See [Upgrade to ece-tools](#upgrade-to-ece-tools) to start using `magento/ece-tools` in 2.1 instead of `magento/magento-cloud-configuration`.
+1.  On your local workstation, create a branch from the Integration environment.
```

Comment:
Yeah, this is a little difficult to grasp, but all development (feature) branches you create in pro must be in the integration environment. Starter is that way anyway. Anyway, I think it could be better...in a Git sort of way. I think I will change it...